setup.py

```diff
@@ -138,6 +139,10 @@ def build_libs(libs):
         my_env["CUDNN_LIBRARY"] = CUDNN_LIBRARY
         my_env["CUDNN_INCLUDE_DIR"] = CUDNN_INCLUDE_DIR
 
+    if WITH_DISTRIBUTED and (WITH_IB_DEVICES or
```

Comment:
Can you make this behave more like all our other setup helpers? The env for IB_DEVICES should be checked inside the helper, and it should also verify that WITH_DISTRIBUTED is enabled.

Reply:
Will do